js/src/api/format/output.js

```diff
@@ -129,12 +129,31 @@ export function outNumber (number) {
   return new BigNumber(number || 0);
 }
 
+export function outPeer (peer) {
```

Comment:
Especially since what we are actually fixing is a bug - for those case need to have the original (doesn't break it) plus the new (this used to break) case.